tests/integration/features/features_int_test.go

```diff
@@ -51,12 +49,12 @@ var _ = Describe("feature preconditions", func() {
 
 			testFeatureName := "test-ns-creation"
 			namespace = envtestutil.AppendRandomNameTo(testFeatureName)
-			dsci = newDSCInitialization(namespace)
+			dsci = fixtures.NewDSCInitialization(namespace)
 		})
 
 		It("should create namespace if it does not exist", func() {
 			// given
-			_, err := getNamespace(namespace)
+			_, err := fixtures.GetNamespace(namespace, envTestClient)
```

Comment:
The convention you can find in k8s golang libraries is to pass ctx,  client and then actual params. I would change the order accordingly.